File: Modul 5/SOAL 4/Bahasa C/PRAK404-2210817210028-Muhammad Aulia Rasyid.c
#include <stdio.h>
int main ()
{
    int a ; float x, y ;
    while(a != 5) {
        printf("Pilih Program : \n");
        printf("1. Penjumlahan \n");
        printf("2. Pengurangan \n");
        printf("3. Perkalian \n");
        printf("4. Pembagian \n");
        printf("5. Exit \n");
        printf("Masukkan Pilihan : "); scanf("%d", &a);
        if (a > 5 || a <= 0){
            printf("Input anda salah, silahkan coba lagi\n\n");}
        else if (a == 5){
            break ; }
        else {
            printf("Masukkan nilai pertama : "); scanf("%f", &x);
            printf("Masukkan nilai kedua : "); scanf("%f", &y);
            if (a == 1){
                printf("Hasil Penjumlahan antara %.2f dengan %.2f adalah %.2f\n\n", x, y, x + y);}
            else if (a == 2){
                printf("Hasil Pengurangan antara %.2f dengan %.2f adalah %.2f\n\n", x, y, x - y);}
            else if (a == 3){
                printf("Hasil Perkalian antara %.2f dengan %.2f adalah %.2f\n\n", x, y, x * y);}
            else {
                printf("Hasil Pembagian antara %.2f dengan %.2f adalah %.2f\n\n", x, y, x / y);} }}
    printf("\nTerimakasih, telah menggunakan kalkulator Rasyid\n");
}

File: Modul 5/SOAL 4/Bahasa Python/PRAK404-2210817210028-Muhammad Aulia Rasyid.py
pil = 1
while pil == 1 :
    print("Pilih Program : ")
    print("1. Penjumlahan ")
    print("2. Pengurangan ")
    print("3. Perkalian ")
    print("4. Pembagian ")
    print("5. Exit ")
    print("Masukkan Pilihan : ", end='') ; a = int(input())
    if a > 5 or a < 1 : 
        print("Input anda salah, silahkan coba lagi\n")
    elif a == 5 : break
    else : 
        print("Masukkan Nilai Pertama : ", end='') ; x = float(input())
        print("Masukkan Nilai Kedua : ", end='') ; y = float(input())
        if a == 1 : print("Hasil Penjumlahan antara %.2f dengan %.2f adalah %.2f\n"%(x, y, x + y))
        elif a == 2 : print("Hasil Pengurangan antara %.2f dengan %.2f adalah %.2f\n"%(x, y, x - y))
        elif a == 3 : print("Hasil Perkalian antara %.2f dengan %.2f adalah %.2f\n"%(x, y, x * y))
        else : print("Hasil Pembagian antara %.2f dengan %.2f adalah %.2f\n"%(x, y, x / y))
print("\nTerimakasih, telah menggunakan kalkulator Rasyid\n")